### PR TITLE
Fix the version & prevent double loading of angular script

### PR DESCRIPTION
**Pull request by ericfong**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,13 +15,19 @@
   },
   "license": "MIT",
   "dependencies": {
-      "node-schedule": "",
-      "request": ""
+    "node-schedule": "0.1.13",
```
Comment (yannisgu) [inline package.json]: Does this fix #9?

File: index.js [hunk 2]
```diff
@@ -133,9 +133,9 @@ Jobs.prototype.runScript = function(file, callback) {
             domain.request = request;
             domain.require = require;
 
-            var ctx = {};
+            var ctx = { req:{session:{isRoot:true}, isRoot:true} };
```
Comment (yannisgu) [inline index.js]: This makes the scheduled scripts running as root, right?

Comment (ericfong) [inline index.js]: Yes

File: package.json (hunk 1 above)
Comment (ericfong) [inline package.json]: Yes. But I think PR #8 also good. 

But please spec the version of deps

**Comment by yannisgu** [comment]
Version fixed by #8, others are in #6

**Pull request closed without merging**